src/app/components/iterations-modal/iterations-modal.component.ts

```diff
@@ -360,25 +376,48 @@ export class FabPlannerIterationModalComponent implements OnInit, OnDestroy, OnC
           this.iterationService.updateIteration(this.iteration)
             .subscribe((iteration) => {
               this.onSubmit.emit(iteration);
+              let toastIterationName = this.iteration.attributes.name;
+              if (toastIterationName.length > 15) {
+                toastIterationName = toastIterationName.slice(0, 15) + '...';
```

Comment:
This is a good workaround, but isn't it better to manage the overflow ellipsis in CSS?

Pseudo-code:
```html
<notification-container>
  <!-- overflow:ellipsis applied on this iteration-name element -->
  <iteration-name title/tooltip="iteration-name">iteration-name</iteration-name>
  has started...
</notification-container>
```

We're already crafting the DOM in next block, so why not just do it there, right?